SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialWorkerConnection.h

```diff
@@ -89,6 +89,7 @@ class SPATIALGDK_API USpatialWorkerConnection : public UObject, public FRunnable
 
 	bool TrySetupConnectionConfigFromCommandLine(const FString& SpatialWorkerType);
 	void SetupConnectionConfigFromURL(const FURL& URL, const FString& SpatialWorkerType);
+	void RequestDeploymentLoginTokens();
```

Comment:
Possibly can be made a const function?

Reply:
Because of some logic in this function, I can not set it to be a  const function. `Worker_Alpha_LoginTokensResponseFuture_Get`